test/golden_tsickle_test.ts

```diff
@@ -6,7 +6,9 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {expect} from 'chai';
+import {expect, use as chaiUse} from 'chai';
+// tslint:disable-next-line:no-require-imports chai-diff needs a CommonJS import.
```

Comment:
How come?  You can
import * as chaiDiff from 'chai-diff';
and then use it as a function.

Reply:
There's a wontfix TypeScript issue about it:
https://github.com/Microsoft/TypeScript/issues/5073